src/site/content/en/blog/origin-agent-cluster/index.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title: Requesting performance isolation with the Origin-Agent-Cluster header
+subhead: A new HTTP response header to limit domain-wide scripting and request dedicated resources from the browser.
+authors:
+  - domenic
+description: >
+  The Origin-Agent-Cluster header cuts off synchronous access to other origins on the same domain, and hints to the browser to give your origin dedicated resources.
+date: 2021-01-11
+hero: hero.jpg
+alt: A cluster of grapes
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - performance
+  - memory
+---
+
```

Comment:
Please break all lines at 100 characters. If you're using VS Code, [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) makes it pretty easy to do. The [Docs To X](https://gsuite.google.com/u/0/marketplace/app/docs_to_x/243920501110?hl=en&pann=docs_addon_widget) Google Docs Add-On also supports this.

Reply:
Done!